tests/test_venue_request_v2.py

```diff
@@ -2237,206 +2226,180 @@ def test_post_decision_stage(self, client, test_client, selenium, request_page,
-    #             'remove_review_form_options': 'title',
-    #             'email_program_chairs_about_reviews': 'Yes, email program chairs for each review received'
-    #         },
-    #         forum=venue['request_form_note'].forum,
-    #         invitation='{}/-/Request{}/Review_Stage'.format(venue['support_group_id'], venue['request_form_note'].number),
-    #         readers=['{}/Program_Chairs'.format(venue['venue_id']), venue['support_group_id']],
-    #         referent=venue['request_form_note'].forum,
-    #         replyto=venue['request_form_note'].forum,
-    #         signatures=['~SomeFirstName_User1'],
-    #         writers=[]
-    #     ))
-    #     assert review_stage_note
-    #     helpers.await_queue()
-
-    #     invitation = openreview_client.get_invitation('V2.cc/2030/Conference/Submission1/-/Official_Review')
-    #     assert len(invitation.edit['note']['readers']) == 1
-    #     assert 'everyone' in invitation.edit['note']['readers']
-    #     assert len(invitation.edit['note']['nonreaders']) == 0
-
-    #     notes = openreview_client.get_notes(invitation='V2.cc/2030/Conference/Submission1/-/Official_Review')
-    #     assert notes[0].readers == ['everyone']
-        
+        submissions = openreview_client.get_notes(invitation='{}/-/Submission'.format(venue['venue_id']), sort='number')
+        assert submissions and len(submissions) == 3
+
+        # Post an official comment stage note
+        now = datetime.datetime.utcnow()
+        start_date = now - datetime.timedelta(days=2)
+        end_date = now + datetime.timedelta(days=3)
+        comment_stage_note = test_client.post_note(openreview.Note(
+            content={
+                'commentary_start_date': start_date.strftime('%Y/%m/%d'),
+                'commentary_end_date': end_date.strftime('%Y/%m/%d'),
+                'participants': ['Program Chairs', 'Assigned Senior Area Chairs', 'Assigned Area Chairs', 'Assigned Reviewers', 'Authors', 'Public (non-anonymously)'],
+                'email_program_chairs_about_official_comments': 'Yes, email PCs for each official comment made in the venue'
+
+            },
+            forum=venue['request_form_note'].forum,
+            invitation='{}/-/Request{}/Comment_Stage'.format(venue['support_group_id'], venue['request_form_note'].number),
+            readers=['{}/Program_Chairs'.format(venue['venue_id']), venue['support_group_id']],
+            referent=venue['request_form_note'].forum,
+            replyto=venue['request_form_note'].forum,
+            signatures=['~SomeFirstName_User1'],
+            writers=[]
+        ))
+        assert comment_stage_note
+        helpers.await_queue()
+
+        process_logs = client.get_process_logs(id=comment_stage_note.id)
+        assert len(process_logs) == 1
+        assert process_logs[0]['status'] == 'ok'
+
+        # Assert that official comment invitations are available
```

Comment:
@melisabok  
Running the comment stage (or any other stage) after having run the `post_decision_stage()` does not create the necessary invitations or changes to invitations since the venueid has changed and we are not getting any submissions with `venue.get_submissions()`

Not sure how we can fix this. We won't be able to release reviews to the public for example once we run the `post_decision_stage()`

Reply:
so how would we get the submissions after the post_decision_stage? 

Using venue_id == accepted and venue_id == rejected instead of venue_id = submitted

Reply:
can we query the three scenarios when we do get_submissions?

Reply:
that should work when running stage where submissions are under review and when submissions where already accepted/rejected.

Reply:
maybe we can ask @carlosmondra to support a query that receives more than one venueid as parameter:

```
GET /notes?content.venueid=TMLR/Submission,TMLR,TMLR/Rejected
```